test/compose.integration.spec.ts

```diff
@@ -81,6 +81,102 @@ describe('compose.integration.spec.ts', () => {
     component.dispose();
   });
 
+  it('works with determineActivationStrategy() - replace', async () => {
+    const { component, compose } = await bootstrapCompose(
+      `<compose view-model.bind="viewModel"></compose>`,
+      {
+        viewModel: class {
```

Comment:
just a note: it can also be an object, i.e: instance of this anonymous class: `new class { ...}`